src/js/pages/jobs/JobsTable.js

```diff
@@ -73,21 +125,50 @@ class JobsTable extends React.Component {
           className="headline table-cell-value flex-box flex-box-col"
           params={{id}}>
           <span className="text-overflow">
-            {job.getName()}
+            {job.name}
           </span>
         </Link>
       </div>
     );
   }
 
+  renderLastRunStatusColumn(prop, row) {
+    let value = row[prop];
+    let statusClasses = classNames({
+      'text-success': value === 'Success',
+      'text-danger': value === 'Failed'
+    });
+
+    return <span className={statusClasses}>{value}</span>;
+  }
+
+  renderStatusColumn(prop, row) {
+    let value = row[prop];
+    let statusClasses = classNames({
+      'text-muted': value === 'completed',
+      'text-color-white': value !== 'completed'
+    });
+    let statusText = null;
+
+    if (value === 'active') {
```

Comment:
Could we create a map and look up the text for this in stead of these `if` - `else of`s?